docs/ops/shape/Unsqueeze_1.md

```diff
@@ -10,23 +10,23 @@
 
 **Inputs**:
 
-*   **1**: Multidimensional input tensor of type *T*. *Required*.
+*   **1**: Multidimensional input tensor of type *T*. **Required**.
```

Comment:
Are there any requirements to the shape? If not then we usally write 'of type *T* and arbitrary shape'.

Reply:
Done